### PR TITLE
Optimize group member count query

### LINKED ISSUE #2327: Group member count query showing fatal error when group contains thousands of members.
**Describe the bug**
Navigate to the individual group and it will show Fatal error: Allowed memory size.

**To Reproduce**
1. Go to any group 
2. See error, 'allowed memory size'.

**Screenshots**
https://www.loom.com/share/6cf42a0d8403491a8fdf42bf73485caf

**Support ticket links**
https://secure.helpscout.net/conversation/1431957224/126343/

### PR DESCRIPTION
**Pull request by asaquzzamanbb**
### All Submissions:

* [ ] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [ ] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes #2327 .

### How to test the changes in this Pull Request:
Navigate to the individual group and it will show Fatal error: Allowed memory size.

1. Go to any group
2. See error, 'allowed memory size'.

### Proof Screenshots or Video

https://prntscr.com/11aka6t

### Other information:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

File: src/bp-templates/bp-nouveau/includes/template-tags.php [hunk 1]
```diff
@@ -1274,11 +1274,11 @@ function bp_nouveau_nav_has_count() {
 	if ( 'directory' === $bp_nouveau->displayed_nav && isset( $nav_item->count ) ) {
 		$count = $nav_item->count;
 	} elseif ( 'groups' === $bp_nouveau->displayed_nav && 'members' === $nav_item->slug ) {
-		$members = groups_get_group_members( array( 'group_id' => bp_get_current_group_id(), 'exclude_admins_mods' => false ) );
+		//$members = groups_get_group_members( array( 'group_id' => bp_get_current_group_id(), 'exclude_admins_mods' => false ) );
```
Comment (KartikSuthar) [inline src/bp-templates/bp-nouveau/includes/template-tags.php:1277]: @asaquzzamanbb Please remove the commented code.

**Review by KartikSuthar — CHANGES_REQUESTED** [review]

File: src/bp-templates/bp-nouveau/includes/template-tags.php (hunk 1 above)
Comment (asaquzzamanbb) [reply src/bp-templates/bp-nouveau/includes/template-tags.php]: Removed here https://github.com/buddyboss/buddyboss-platform/pull/2328/commits/981e7eb0fd2cf7441f226e3fdec58950ac8f2e07#diff-bb67106ac2f5afb6d92064a4efc271bd82a327c43a8a1d05db81491bfd3ca3bb

**Review by KartikSuthar — DISMISSED** [review]
@asaquzzamanbb Code looks good to me.

**Comment by asaquzzamanbb** [comment]
Merge with dev branch https://github.com/buddyboss/buddyboss-platform/commit/47ba314a543be919bff24abbbe8d1ccf21ccaf89

**Comment by asaquzzamanbb** [comment]
@sumitbuddyboss 

The issue you have mention in video is not related of this PR yet. But i have fixed it in this PR. 
https://github.com/buddyboss/buddyboss-platform/pull/2328/commits/424620c701f43a34c4b613b093d86b46ee9556d3

**Review by KartikSuthar — APPROVED** [review]

**Comment by sumitbuddyboss** [comment]
Hello @asaquzzamanbb 

Fixed > I have checked and it's working fine from my end for the proof please check the below video.

Video:
https://www.loom.com/share/48d4a82007bb48d39246ae05d6b67167

Note:
Getting issue In group > Send message tab so for that @asaquzzamanbb  has created a new issue for that.
https://github.com/buddyboss/buddyboss-platform/issues/2397

Ready for release.

cc @sourabhmatolia @chetansatasiya 

Thank you

**Pull request merged**